developer-support/release-notes/pump.mdx

```diff
@@ -17,6 +17,64 @@ Our minor releases are supported until our next minor comes out.
 
 ## 1.13 Release Notes
 
+### 1.13.2 Release Notes
+
+#### Release Date 16 December 2025
+
+#### Release Highlights
+
+In this patch release, we've resolved SQL Pump schema migration issues for sharded tables. For a comprehensive list of changes, please refer to the detailed [changelog](#Changelog-v1.13.2).
+
+#### Breaking Changes
+
```

Comment:
Can we add a "no breaking changes" note? It should point out that there's potential for a start-up delay if schema migration is enabled (directing them to the change log entry).